2024-04-08 08:26:06,833 INFO    MainThread:465542 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-08 08:26:06,834 INFO    MainThread:465542 [wandb_setup.py:_flush():76] Configure stats pid to 465542
2024-04-08 08:26:06,834 INFO    MainThread:465542 [wandb_setup.py:_flush():76] Loading settings from /home/russellsb/.config/wandb/settings
2024-04-08 08:26:06,834 INFO    MainThread:465542 [wandb_setup.py:_flush():76] Loading settings from /mnt/c/Users/rsamm/root/repos/Data Science for Business Package/4. Operations Department/wandb/settings
2024-04-08 08:26:06,835 INFO    MainThread:465542 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-08 08:26:06,835 INFO    MainThread:465542 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-08 08:26:06,835 INFO    MainThread:465542 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-08 08:26:06,836 INFO    MainThread:465542 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-08 08:26:06,837 INFO    MainThread:465542 [wandb_init.py:_log_setup():521] Logging user logs to /mnt/c/Users/rsamm/root/repos/Data Science for Business Package/4. Operations Department/wandb/run-20240408_082606-o8dovpqv/logs/debug.log
2024-04-08 08:26:06,837 INFO    MainThread:465542 [wandb_init.py:_log_setup():522] Logging internal logs to /mnt/c/Users/rsamm/root/repos/Data Science for Business Package/4. Operations Department/wandb/run-20240408_082606-o8dovpqv/logs/debug-internal.log
2024-04-08 08:26:06,838 INFO    MainThread:465542 [wandb_init.py:_jupyter_setup():467] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f43a3449840>
2024-04-08 08:26:06,839 INFO    MainThread:465542 [wandb_init.py:init():561] calling init triggers
2024-04-08 08:26:06,839 INFO    MainThread:465542 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-04-08 08:26:06,839 INFO    MainThread:465542 [wandb_init.py:init():611] starting backend
2024-04-08 08:26:06,840 INFO    MainThread:465542 [wandb_init.py:init():615] setting up manager
2024-04-08 08:26:06,847 INFO    MainThread:465542 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-04-08 08:26:06,850 INFO    MainThread:465542 [wandb_init.py:init():623] backend started and connected
2024-04-08 08:26:06,871 INFO    MainThread:465542 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2024-04-08 08:26:06,874 INFO    MainThread:465542 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-04-08 08:26:06,874 INFO    MainThread:465542 [wandb_init.py:init():715] updated telemetry
2024-04-08 08:26:06,912 INFO    MainThread:465542 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2024-04-08 08:26:07,399 INFO    MainThread:465542 [wandb_run.py:_on_init():2357] communicating current version
2024-04-08 08:26:07,521 INFO    MainThread:465542 [wandb_run.py:_on_init():2366] got version response 
2024-04-08 08:26:07,522 INFO    MainThread:465542 [wandb_init.py:init():799] starting run threads in backend
2024-04-08 08:26:07,843 INFO    MainThread:465542 [wandb_run.py:_console_start():2335] atexit reg
2024-04-08 08:26:07,844 INFO    MainThread:465542 [wandb_run.py:_redirect():2190] redirect: wrap_raw
2024-04-08 08:26:07,844 INFO    MainThread:465542 [wandb_run.py:_redirect():2255] Wrapping output streams.
2024-04-08 08:26:07,845 INFO    MainThread:465542 [wandb_run.py:_redirect():2280] Redirects installed.
2024-04-08 08:26:07,846 INFO    MainThread:465542 [wandb_init.py:init():842] run started, returning control to user process
2024-04-08 08:29:51,551 INFO    MainThread:465542 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-08 08:29:51,551 INFO    MainThread:465542 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-08 08:30:54,627 INFO    MainThread:465542 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-08 08:30:54,662 INFO    MainThread:465542 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-08 08:30:54,663 INFO    MainThread:465542 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-08 08:30:56,616 INFO    MainThread:465542 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-08 08:30:56,636 INFO    MainThread:465542 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-08 08:30:56,637 INFO    MainThread:465542 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-08 08:30:58,877 INFO    MainThread:465542 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-08 08:30:58,885 INFO    MainThread:465542 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-08 08:30:58,885 INFO    MainThread:465542 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-08 08:31:01,902 INFO    MainThread:465542 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-08 08:31:01,904 INFO    MainThread:465542 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-08 08:31:01,904 INFO    MainThread:465542 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-08 08:31:05,675 INFO    MainThread:465542 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-08 08:31:05,804 INFO    MainThread:465542 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-08 08:31:05,804 INFO    MainThread:465542 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-08 08:31:07,924 INFO    MainThread:465542 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-08 08:31:07,992 INFO    MainThread:465542 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-08 08:31:07,992 INFO    MainThread:465542 [wandb_setup.py:_flush():76] Configure stats pid to 465542
2024-04-08 08:31:07,993 INFO    MainThread:465542 [wandb_setup.py:_flush():76] Loading settings from /home/russellsb/.config/wandb/settings
2024-04-08 08:31:07,993 INFO    MainThread:465542 [wandb_setup.py:_flush():76] Loading settings from /mnt/c/Users/rsamm/root/repos/Data Science for Business Package/4. Operations Department/wandb/settings
2024-04-08 08:31:07,994 INFO    MainThread:465542 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-08 08:31:07,996 INFO    MainThread:465542 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-08 08:31:07,997 INFO    MainThread:465542 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-08 08:31:07,998 INFO    MainThread:465542 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-08 08:31:07,999 INFO    MainThread:465542 [wandb_init.py:_log_setup():521] Logging user logs to /mnt/c/Users/rsamm/root/repos/Data Science for Business Package/4. Operations Department/wandb/run-20240408_083107-nz2063k1/logs/debug.log
2024-04-08 08:31:08,000 INFO    MainThread:465542 [wandb_init.py:_log_setup():522] Logging internal logs to /mnt/c/Users/rsamm/root/repos/Data Science for Business Package/4. Operations Department/wandb/run-20240408_083107-nz2063k1/logs/debug-internal.log
2024-04-08 08:31:08,000 INFO    MainThread:465542 [wandb_init.py:init():561] calling init triggers
2024-04-08 08:31:08,001 INFO    MainThread:465542 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-04-08 08:31:08,002 INFO    MainThread:465542 [wandb_init.py:init():586] re-initializing run, found existing run on stack: o8dovpqv
2024-04-08 08:31:08,006 INFO    MainThread:465542 [wandb_run.py:_finish():2064] finishing run russellsb/x-ray-chest-disease-classification/o8dovpqv
2024-04-08 08:31:08,008 ERROR   MainThread:465542 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-04-08 08:31:08,009 INFO    MainThread:465542 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-08 08:31:08,009 INFO    MainThread:465542 [wandb_init.py:_jupyter_teardown():449] cleaning up jupyter logic
2024-04-08 08:31:08,010 INFO    MainThread:465542 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2024-04-08 08:31:08,011 INFO    MainThread:465542 [wandb_run.py:_restore():2287] restore
2024-04-08 08:31:08,012 INFO    MainThread:465542 [wandb_run.py:_restore():2293] restore done
2024-04-08 08:31:14,230 INFO    MainThread:465542 [wandb_run.py:_footer_history_summary_info():3936] rendering history
2024-04-08 08:31:14,231 INFO    MainThread:465542 [wandb_run.py:_footer_history_summary_info():3968] rendering summary
2024-04-08 08:31:14,244 INFO    MainThread:465542 [wandb_run.py:_footer_sync_info():3895] logging synced files
